backend/src/main/kotlin/be/osoc/team1/backend/entities/Student.kt

```diff
@@ -130,26 +135,58 @@ class Student(
 
     @ManyToMany(cascade = [CascadeType.MERGE])
     @OrderBy
+    @field:JsonView(StudentView.Full::class)
     val skills: Set<Skill> = sortedSetOf(),
+
+    @field:JsonView(StudentView.Basic::class)
     val alumn: Boolean = false,
+
+    @field:JsonView(StudentView.Full::class)
     val possibleStudentCoach: Boolean = false,
+
+) {
+
     @OneToMany(cascade = [CascadeType.ALL], orphanRemoval = true)
+    @field:JsonView(StudentView.Full::class)
     @JsonSerialize(using = AnswerListSerializer::class)
-    val answers: List<Answer> = listOf()
-) {
+    var answers: List<Answer> = listOf()
 
     @Id
+    @field:JsonView(StudentView.Basic::class)
     val id: UUID = UUID.randomUUID()
 
+    @field:JsonView(StudentView.Basic::class)
     var status: StatusEnum = StatusEnum.Undecided
 
     @OneToMany(cascade = [CascadeType.ALL], orphanRemoval = true)
+    @field:JsonView(StudentView.Full::class)
     @JsonSerialize(using = StatusSuggestionListSerializer::class)
     val statusSuggestions: MutableList<StatusSuggestion> = mutableListOf()
 
     @OneToMany(cascade = [CascadeType.ALL], orphanRemoval = true)
+    @field:JsonView(StudentView.Full::class)
     @JsonSerialize(using = CommunicationListSerializer::class)
     val communications: MutableList<Communication> = mutableListOf()
+
+    @JsonGetter("statusSuggestionsPercentage")
+    fun calculateStatusSuggestionPercentage(): Map<SuggestionEnum, Int> = statusSuggestions.groupingBy { it.status }.eachCount()
```

Comment:
The name here is misleading, this returns the amount of yes/maybe/no suggestions instead of percentages. I'm not sure which one is more convenient for the frontend.

Reply:
Ah jes, my intent was for them to be percentages, I forgot about that part. @Mikxox @NielsPraet, Do you guys want % or #?

Reply:
# would be preferred

Reply:
In that case could you just update the name of the function and the value of the annotation @lars-vc ?